commerce-product-content-category-web/src/main/resources/com/liferay/commerce/product/content/category/web/internal/portlet/template/dependencies/portlet_display_template_basic.ftl

```diff
@@ -1,12 +1,26 @@
 <div class="row">
 	<#assign
-		image = cpCategoryContentDisplayContext.getDefaultImageSrc(themeDisplay)
+		image = ''
 
-		title = assetCategory.getTitle(locale)
+		title = ''
 
-		description = assetCategory.getDescription(locale)
+		description = ''
 	/>
 
+	<#if cpCategoryContentDisplayContext.getDefaultImageSrc(themeDisplay)??>
+		<#assign
+			image = cpCategoryContentDisplayContext.getDefaultImageSrc(themeDisplay)
+		/>
+	</#if>
+
+	<#if assetCategory??>
+		<#assign
+			title = assetCategory.getTitle(locale)
+
+			description = assetCategory.getTitle(locale)
```

Comment:
@marco-leo @Commando18 should this still be `description = assetCategory.getDescription(locale)`?